### ISSUE TITLE
Linter proposal: disallow `visibility: visible`

### ISSUE THREAD

**Issue by janpaul123** [brigade/scss-lint#733]
And encourage to use `visibility: inherit` instead, because using `visibility: visible` may unintentionally override an inherited `visibility: hidden`. And in the rare cases where it is actually intentionally, I think it's good have an explicit comment that disables this rule.

**Comment by srawlins** [comment]
I like this. I think it could be a generic DisallowedValues linter or something:

```yaml
DisallowedValues:
    enabled: true
    blacklist:
      - property: visibility
        value: visible
        reason: Use "visibility: inherit" instead, to avoid accidentally overriding "visibility: hidden."
      - property: font-family
        value: serif
        reason: We don't use serif.
```

The YAML is getting a little deep... but I think that list-of-dictionaries syntax works...

**Comment by janpaul123** [comment]
Ah yes, that sounds like a sensible abstraction!

**Comment by lencioni** [comment]
What do you think about calling it `disallowed_value: serif` instead of `value: serif`? I don't have strong feelings about it, but I think that `disallowed_value` might be a little clearer.

Alternatively, what about combining `property` and `value` like:

```yaml
DisallowedValues:
    enabled: true
    blacklist:
      - style: visibility: visible
        reason: Use "visibility: inherit" instead, to avoid accidentally overriding "visibility: hidden."
```
